### ISSUE TITLE
Self-update issue

### ISSUE THREAD

**Issue by sergeyklay** [Codeception/Codeception#2038]
```sh
$ sudo codecept self-update
Codeception version 2.0.14

Checking for a new version...

A newer version is available: 2.1.0-beta

Do you want to update? y

Updating...

/usr/local/bin/codecept has been updated.

$ sudo codecept --version
Codeception version 2.0.14
```

**Comment by sergeyklay** [comment]
```
$ ls -l /usr/local/bin/codecept
-rwxr-xr-x 1 root root 1292559 Jun 11 19:43 /usr/local/bin/codecept
```

**Comment by Naktibalda** [comment]
There is an inconsistency between what self-update checks and what it downloads.
It checks what is the latest tag in github, but downloads http://codeception.com/codecept.phar

**Comment by Naktibalda** [comment]
Have you fixed it?

**Comment by DavertMik** [comment]
I think we don't have beta versions now. 
And I think we won't have betas in phar.

**Comment by Naktibalda** [comment]
It's better to prepare for beta versions if you are going to publish 2.2.0-beta or 3.0.0-beta.

**Comment by DavertMik** [comment]
I think they can be downloaded via Composer only. Why not? Phar should be for stable environments.

**Comment by DavertMik** [comment]
You can't do self-revert after self-update :)

**Comment by Naktibalda** [comment]
http://codeception.com/builds

**Comment by DavertMik** [comment]
Yes, I know we had beta-versions and you can download older versions. But I don't see a reason to provide packages for beta versions, as majority of our users (ok, I didn't count them actually) can download it with Composer

**Comment by Naktibalda** [comment]
> $ php codecept.phar self-update
> Codeception version 2.1.4
> 
> Checking for a new version...
> 
> A newer version is available: 2.2.0-RC
> 
> Do you want to update? y
> 
> Updating...
> 
> codecept.phar has been updated.
> 
> $ php codecept.phar self-update
> Codeception version 2.1.8
> 
> Checking for a new version...
> 
> A newer version is available: 2.2.0-RC
